packages/plugin/src/generators/create-package/create-package.ts

```diff
@@ -56,6 +70,20 @@ export async function createPackageGenerator(
     await formatFiles(host);
   }
 
+  if (options.isTsSolutionSetup) {
+    const projectPackageManagerWorkspaceState =
+      getProjectPackageManagerWorkspaceState(host, options.projectRoot);
+
+    if (projectPackageManagerWorkspaceState !== 'included') {
+      tasks.push(
+        getProjectPackageManagerWorkspaceStateWarningTask(
```

Comment:
Maybe a later enhancement, but if we know that the project is missing from workspaces then we could add the entry for them automatically. We had the same idea for `nx import` when the entry is missing.

Reply:
Yeah, we discussed this at the beginning and agreed on logging a warning for now. I'll create a separate task to add the entry automatically.